src/main/scala/org/apache/spark/streamdm/classifiers/trees/HoeffdingTree.scala

```diff
@@ -175,7 +181,7 @@ class HoeffdingTreeModel(val espec: ExampleSpecification, val numericObserverTyp
   var blockNumExamples: Int = 0
 
   var lastExample: Example = null
-
+  var listExamples: ArrayBuffer[Example] = new ArrayBuffer[Example]()
```

Comment:
It would be for the best if you also included a comment in here explaining the purpose of the `listExamples` var.